release-notes/release-notes58.md

```diff
@@ -188,14 +188,21 @@ This section describes issues that users of {{site.productname}} 5.8 may encount
 
 **Outline**
 
-- [](#)
+- [Unexpected behavior when moving the caret using the PageDown key](#unexpectedbehaviorwhenmovingthecaretusingthepagedownkey)
+- [Accessibility when using keyboard navigation to resize the editor](#accessibilitywhenusingkeyboardnavigationtoresizetheeditor)
 
-###
+### Unexpected behavior when moving the caret using the PageDown key
```

Comment:
This issue applies to both the `PageDown` and `End` key so we'll need to include that.